🜚 .xml

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ManageBusinessInteractionResponse xmlns="http://schemas.dgit.biz/telflow/cim/v3">
+    <User>
+        <ID>PTR000000000214</ID>
+    </User>
+    <BusinessInteraction xsi:type="CustomerOrder" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <ID>ORD000000008610</ID>
+        <SpecifiedBy>
+            <ID>BOD-SPC000000000061</ID>
+            <Name>BoD P2P New Order</Name>
+            <Version>1.0</Version>
+            <Process>New-Order-v1-0</Process>
+        </SpecifiedBy>
+        <ContractSummary>
+            <ContractStartDate>2018-11-27T13:00:00Z</ContractStartDate>
+            <ContractEndDate>2018-12-07T13:00:00Z</ContractEndDate>
+        </ContractSummary>
+        <InteractionStatus>
```

Comment:
InteractionStatus? Really? In this economy?

Reply:
blame marketing